src/Inventory.js

```diff
@@ -7,6 +7,7 @@ class Inventory extends Map {
   addItem(item) {
     this.set(item.uuid, item);
     item.isHeld = true;
+    item.container = this;
```

Comment:
Hmm. This was mainly a solution to the problem of having the `Inventory` hold a reference to an item that may have already decayed. I am not sure how else this would be solved, though it could just have a reference back to the containing item then decay could call `this.container.inventory.removeItem`